test/helpers.py

```diff
@@ -216,6 +216,12 @@ def add_field_data(mesh, value, dtype):
     return mesh2
 
 
+def add_node_sets(mesh):
+    mesh2 = copy.deepcopy(mesh)
+    mesh2.node_sets = {"nset": numpy.array([1, 2])}
```

Comment:
Does it have to be called `"nset"`. Perhaps something completely different would be appropriate, like `"metal"` or your favorite band.